src/Server.php

```diff
@@ -79,8 +95,8 @@ public function authenticate(string $username, string $password): ConnectionInte
             $this->getServerString(),
             $username,
             $password,
-            0,
-            1,
+            $this->options,
+            $this->retries,
```

Comment:
We need tests: maybe we can try the `CL_EXPUNGE` flag to prove this functionality

Reply:
I'll try to prepare tests during weekend to prove this changes actually works